tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -67,6 +67,12 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="TracerSettings.TraceEnabled"/>
         public const string TraceEnabled = "DD_TRACE_ENABLED";
 
+        /// <summary>
+        /// Configuration key for enabling Standalone ASM, thus disabling APM tracing and its billing.
+        /// Default is value is false (disabled).
+        /// </summary>
+        public const string ExperimentalAppsecStandaloneEnabled = "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED";
```

Comment:
For future reference, the C# property and parameter names don't need to match the env var exactly. For this one, for example, I would've been ok with something shorter and easier like `StandaloneAsmEnabled`. But I'm not asking to go back and change it all now that it's done, especially since it's a temporary setting and there is no public C# api (see my other comment about removing the public C# property).

Reply:
I agree, we should just drop the "Experimental" prefix 🙂

Reply:
Okay :til:
I've changed it 😄 (not that difficult with a search and replace)